serverLib.sml

```diff
@@ -499,44 +519,63 @@ structure ReadJSON = struct
       ,("committedDate", transform with_date read_date)
       ] empty_obj
 
+  val read_label : string basic_reader = read_dict
+    [("name", replace_acc read_string)] ""
+  val read_labels = read_dict
+    [("nodes", read_list read_label)] []
+
   val read_pr : pr option basic_reader =
     read_dict
       [("mergeable", transform mergeable_only read_string)
       ,("number", transform (Option.map o with_num) read_number)
       ,("headRefName", transform (Option.map o with_head_ref) read_string)
+      ,("labels", transform (Option.map o with_labels) read_labels)
       ,("headRef",
         read_dict
           [("target", transform (Option.map o with_head_obj) read_obj)])]
       (SOME empty_pr)
 
 end
 
+val no_test_labels = ["test failing", "no test"]
+fun is_no_test_label l = List.exists (fn l2 => l2 = l) no_test_labels
```

Comment:
use `(equal l)` instead of the lambda

Reply:
Can't. Not in scope. What signature is it in?

Reply:
Hmm, it should be in `utilLib` - surprised that's not already open (it seems like it should be transitively via `apiLib`)

Reply:
Nevermind, it was in scope in the file, just not in the REPL.